.github/workflows/deployminder.yml

```diff
@@ -11,8 +11,6 @@ jobs:
   remind:
     if: github.event.pull_request.merged == true
     runs-on: ubuntu-latest
-    permissions:
-        issues: write
     steps:
       - name: Comment on PR
         uses: actions/github-script@v7
```

Comment:
maybe should revert to v6

Reply:
ah, was it working before update?

Reply:
Doesn't look like we were ever using v6?

Reply:
im not sure anymore :\ this works on a private repo as many times as i can trigger it, but here it fails. both repos have the same perms in settings > actions, so im just confused on what the issue is.

it was working before the permissions add, then it wasn't

Reply:
going off of https://github.com/actions/github-script?tab=readme-ov-file#comment-on-an-issue and https://github.com/litestar-org/litestar/blob/119169d1c335f68cbc4673006b4cb2aa96983513/.github/workflows/pr-merged.yml#L49